workspaces/orchestrator/plugins/orchestrator-backend/src/service/api/mapping/V2Mappings.test.ts

```diff
@@ -143,7 +149,16 @@ describe('scenarios to verify mapToProcessInstanceDTO', () => {
     expect(result.duration).toEqual(duration);
 
     expect(result).toBeDefined();
-    expect(result.status).toEqual(
+    if (!result.state) {
+      throw new Error('result.state should be defined');
+    }
```

Comment:
```suggestion
expect(result.state).toBeDefined();
```

Reply:
this leaves me with a typescript error

Reply:
what error? it seems a standard way to handle this

Reply:
typescript doesn't recognize that this "expect(result.state).toBeDefined()" means it's defined

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/user-attachments/assets/f82e0180-3a75-40e7-9b22-4c148b06246c)

Reply:
is this better?
https://github.com/redhat-developer/rhdh-plugins/pull/211/commits/2e1671d01e914088513e58f88586a0fe3e0ffdca

Reply:
ok so the issue isn't on `expect(result.state).toBeDefined();` but on the following `expect`. 
Now it makes sense and your solution looks good.
Thanks :)